Frontend/frontend/src/Views/Login.js

```diff
@@ -36,10 +36,10 @@ export default class Login extends React.Component {
     } else {
       switch (loginResult.status) {
         case 404:
-          this.setState({ error: "Username does not exist."});
+          this.setState({ error: "Invalid username, email or password."});
           break;
         case 403:
-          this.setState({ error: "Invalid password."});
+          this.setState({ error: "Invalid username, email or password."});
           break;
```

Comment:
Can you log in with an email? If not this should just be Invalid username or password